dbt/sagerx/models/intermediate/orange_book/int_fda_ndc_to_te.sql

```diff
@@ -1,21 +1,22 @@
--- DISCLAIMER: because I can't figure out NDC-level mapping, I only include applications with a single OB TE code
-WITH cte AS (
-    SELECT
+-- DISCLAIMER: This model is under development and incomplete.
+
+with cte as (
+    select
         fda.ndc11
         , obp.te_code
-        , COUNT(fda.ndc11) OVER( PARTITION BY fda.ndc11 ) AS num_te_codes
-    FROM {{ source('orange_book', 'orange_book_products') }} AS obp
-    INNER JOIN {{ ref('stg_fda_ndc__ndcs') }} AS fda 
-        ON concat(CASE WHEN obp.appl_type = 'A' THEN 'ANDA' ELSE 'NDA' END, obp.appl_no) = fda.applicationnumber
-    GROUP BY fda.ndc11, obp.te_code
+        , count(fda.ndc11) over( partition by fda.ndc11 ) as num_te_codes
+    from {{ source('orange_book', 'orange_book_products') }} as obp
```

Comment:
NOTE: since this model is way incomplete, I didn't do this right now, but when we revisit, we need to not have this intermediate model directly access datasource per the style guide.